lib/event_store_client/adapters/grpc/connection.rb

```diff
@@ -26,31 +41,30 @@ def call(stub_klass, options: {})
         )
       end
 
-      private
-
-      attr_reader :cert
-
-      def initialize
-        retries ||= 0
-        @cert =
-          Net::HTTP.start(
-            config.eventstore_url.host, config.eventstore_url.port,
-            use_ssl: true,
-            verify_mode: verify_ssl,
-            &:peer_cert
-          )
-      rescue SocketError
-        sleep config.socket_error_retry_sleep
-        retry if (retries += 1) <= config.socket_error_retry_count
-        raise SocketErrorRetryFailed
-      end
+      def cert
+        retries = 0
```

Comment:
this might inside begin block?

Reply:
If to put it inside begin block, then it will be re-assigned each time `retry` happens